docs/source/examples/simpleflight.py

```diff
@@ -13,14 +14,12 @@
 V_min = Var("V_{min}", 22, "m/s", "takeoff speed")
 C_Lmax = Var("C_{L,max}", 1.5, "-", "max CL with flaps down")
 S_wetratio = Var("(\\frac{S}{S_{wet}})", 2.05, "-", "wetted area ratio")
+W_W_coeff1 = Var("W_{W_{coeff1}}",8.71e-5,"1/m","Wing Weight Coefficent 1")
+W_W_coeff2 = Var("W_{W_{coeff2}}",45.24,"Pa","Wing Weight Coefficent 2")
```

Comment:
@mjburton11, not to be picky, (but that's how life is in the world of GPkit development) -- need spaces after commas in the above